core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/DownloadConnector.java

```
@@ -66,7 +69,11 @@ public Response download(@PathParam("workspace") String workspace,
         fileName = node.getProperty("exo:title").getString();
       }
       // decode the fileName in case the fileName is already encoded, for the old uploaded files.
-      fileName = URLDecoder.decode(fileName, "UTF-8");
+      try {
+        fileName = URLDecoder.decode(fileName, "UTF-8");
+      }catch (Exception e){
+        LOG.info("The fileName is already decoded");
```

Change it to debug, no need to show this information.

---

ok

---

Wouldn't worth also to log the stack trace on the debug module? WDYT @ahamdi

---

No need for a stack trace as we will ignore the exception which is caused by the fact that the filename is already decoded.